<ui version="4.0" >
 <class>AddSetGet</class>
 <widget class="QDialog" name="AddSetGet" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Add get/set methods</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>340</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Create get/set methods for</string>
     </property>
     <property name="checkable" >
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <widget class="QCheckBox" name="setInline" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>inline</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="getInline" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>inline</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="setName" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="set" >
        <property name="text" >
         <string>set method</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="getName" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="get" >
        <property name="text" >
         <string>get method</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AddSetGet</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>296</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel" >
     <x>143</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>get</sender>
   <signal>toggled(bool)</signal>
   <receiver>getName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>45</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>129</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>get</sender>
   <signal>toggled(bool)</signal>
   <receiver>getInline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>24</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>300</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set</sender>
   <signal>toggled(bool)</signal>
   <receiver>setName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel" >
     <x>147</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>set</sender>
   <signal>toggled(bool)</signal>
   <receiver>setInline</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>23</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel" >
     <x>303</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
